tests/examples/shims/client.lua

```diff
@@ -31,6 +31,7 @@ function client.gen_fake(args)
     ret.size_hints = {}
     ret.border_width = 1
     ret.icon_sizes = {{16,16}}
+    ret.name = "client"
```

Comment:
How about `Fake Client` or `Example Client` or something like that? `client` might be a bit too generic.